apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -152,6 +152,20 @@ public static Builder builder() {
     return new RealApolloSubscriptionCall<>(subscription, subscriptionManager);
   }
 
+  /**
+   * Set {@link SubscriptionManager} to a connectible state.
+   */
+  public void startSubscriptions() {
```

Comment:
My concern is regarding naming here. `startSubscriptions` is misleading as first thought that I had by looking at API was impression that all subscriptions that were active before and stopped via `stopSubscriptions ` will be re-started again.

While `stopSubscriptions` is partially correct (except it disables any new subscriptions), as it really stops all active subscriptions.

Not sure if `enableSubscriptions / disableSubscriptions` is better.

@digitalbuddha do you have any ideas how to make this API better?

Reply:
That naming seems good.  Changes pushed.